capture/genBitmaps.js

```diff
@@ -1,251 +1,260 @@
-
 var fs = require('fs');
+var cwd = fs.workingDirectory;
+var __dirname = null;
+var searchString = "genBitmaps.js";
+require('system').args.forEach(function (arg, i) {
+  var position = arg.length - searchString.length;
+  var lastIndex = arg.indexOf(searchString, position);
+  if (lastIndex !== -1 && lastIndex === position) {
+    __dirname = arg.replace(fs.separator + 'genBitmaps.js', '');
+  }
+});
```

Comment:
this looks like duplicated code from 'echoFiles.js', maybe extract it to separate helper file?